core-model/src/main/java/com/sequenceiq/cloudbreak/view/StackView.java

```diff
@@ -184,12 +185,15 @@ default boolean isModificationInProgress() {
         return CREATE_IN_PROGRESS.equals(status)
                 || UPDATE_IN_PROGRESS.equals(status)
                 || STOP_IN_PROGRESS.equals(status)
+                || EXTERNAL_DATABASE_STOP_IN_PROGRESS.equals(status)
                 || START_IN_PROGRESS.equals(status)
+                || EXTERNAL_DATABASE_START_IN_PROGRESS.equals(status)
                 || DELETE_IN_PROGRESS.equals(status);
     }
 
     default boolean isReadyForStart() {
-        return STOPPED.equals(getStatus()) || START_REQUESTED.equals(getStatus()) || START_IN_PROGRESS.equals(getStatus());
+        return STOPPED.equals(getStatus()) || START_REQUESTED.equals(getStatus()) || START_IN_PROGRESS.equals(getStatus())
+                || EXTERNAL_DATABASE_STOP_FINISHED.equals(getStatus());
     }
```

Comment:
Like for `Stack`, I think it would make sense to also include `EXTERNAL_DATABASE_START_IN_PROGRESS` and `EXTERNAL_DATABASE_START_FINISHED` here.

Reply:
ok